test/playlist-loader.test.js

```diff
@@ -655,57 +655,50 @@ QUnit.test('resolveMediaGroupUris resolves media group URIs', function(assert) {
       uri: 'playlist-0'
     }],
     mediaGroups: {
-      ULTRACLEAR: {
-        ULTRACLEAR1: {
-          LOW: {
-            uri: 'ultraclear-1-low-uri'
-          },
-          HIGH: {
-            uri: 'ultraclear-1-high-uri'
-          }
+      // CLOSED-CAPTIONS will never have a URI
+      'CLOSED-CAPTIONS': {
+        cc1: {
+          English: {}
         }
       },
-      AUDIO: {
-        AUDIO1: {
-          LOW: {
-            uri: 'audio-1-low-uri'
-          },
-          HIGH: {
-            uri: 'audio-1-high-uri'
+      'AUDIO': {
+        low: {
+          // audio doesn't need a URI if it is a label for muxed
+          main: {},
+          commentary: {
+            uri: 'audio-low-commentary-uri'
           }
         },
-        AUDIO2: {
-          LOW: {
-            uri: 'audio-2-low-uri'
-          },
-          HIGH: {
-            uri: 'audio-2-high-uri'
+        high: {
+          main: {},
```

Comment:
not necessary, but if you want to cover another edge case, you can have a muxed audio label without a uri in one audio group, and the same labeled audio in another group can have unmuxed so it would have a uri. Playlist loader doesn't care about this detail though, its taken into account in media-groups, but incase you wanted another test case